### ISSUE TITLE
[1.12.2] Crash on respawn (Magiculture 2)

### ISSUE THREAD

**Issue by BLAMM67** [OreCruncher/DynamicSurroundings#328]
*Mod Version:* 3.4.9.9

*Forge Version:* 14.23.2.2624

*Link to crash log (if applicable):* [Crash log](https://pastebin.com/yTDXpANa)

*Description:* Playing on a Magiculture 2 (0.2.2.3) server. Client crashes after player dies when they click Respawn. Happens every time to every player.

**Comment by OreCruncher** [comment]
Link the client log please.  May be other exceptions in the log to give clues.

**Comment by BLAMM67** [comment]
Updated with client log.

**Comment by BLAMM67** [comment]
It's [Magiculture 2](https://minecraft.curseforge.com/projects/magiculture-2). I can try those upgrades and recommend them to the pack author. I'm surprised no one else seems to be have this problem with it though.

**Comment by OreCruncher** [comment]
Yeah.  I would have expected to see an error report sooner for this pack.  This error could crop up when a player dies.  There are other conditions but they would be much more rare.  But in general the pack author should keep up to date with mod updates.

**Comment by OreCruncher** [comment]
@BLAMM67 Have you made progress getting the pack updated?

**Comment by BLAMM67** [comment]
Not yet. I have attempted to make the upgrades myself to get an idea of what is involved. Unfortunately, the need to upgrade Forge forces many of the other mods in the pack to also need an update.
The issue also seems to be inconsistent and hasn't happened as often recently, so we aren't as in a rush to get it fixed.
I'll update here if there is any progress.
